src/storage/src/source/generator.rs

```diff
@@ -212,144 +215,147 @@ fn render_simple_generator<G: Scope<Timestamp = MzOffset>>(
     let (mut data_output, stream) = builder.new_output::<AccountedStackBuilder<_>>();
     let (mut stats_output, stats_stream) = builder.new_output();
 
-    let button = builder.build(move |caps| async move {
-        // Do not run the load generator until we have all of our source
-        // exports. Waiting here is fine because we know that their creation and
-        // scheduling of this dataflow is imminent.
-        //
-        // TODO(#26765): can this limitation be removed?
-        if required_exports != config.source_exports.len() {
-            std::future::pending().await
-        }
+    let busy_signal = Arc::clone(&config.busy_signal);
```

Comment:
Is this entire code untested? Seems very complex

Reply:
This code runs every time a load gen source is created so it's probably a false positive from the coverage report